src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -957,13 +942,15 @@ def visit_Index(self, node: ast.Index):
 
     def visit_Subscript(self, node: ast.Subscript):
         assert isinstance(node.ctx, (ast.Load, ast.Store))
-        index = self.visit(node.slice)
+        index_asts = (
+            node.slice.value.elts if isinstance(node.slice.value, ast.Tuple) else node.slice.value
```

Comment:
Note: This will fail in Python 3.9. `node.slice` can be a `ast.Tuple` itself, which does not have a `value` attribute.

Reply:
Should be resolved now.